site/docs/master/velero-install.md

```diff
@@ -0,0 +1,44 @@
+# Velero Install CLI
+
+This document serves as a guide to using the `velero install` CLI command to install `velero` server components into your kubernetes cluster.
+
+_NOTE_: `velero install` will, by default, use the version information to determine the version of the server compoents to deploy. This behavior may be overridden by using the `--image` flag. Refer to [Building Server Component Container Images][1].
+
+## Usage
+
+This section explains some of the basic flags supported by the `velero install` CLI command. For a complete explaination of the flags, please run `velero install --help`
+
+```bash
+velero install \
+    --plugins <PLUGIN_CONTAINER_IMAGE [PLUGIN_CONTAINER_IMAGE]>
+    --provider <YOUR_PROVIDER> \
+    --bucket <YOUR_BUCKET> \
+    --secret-file <PATH_TO_FILE> \
+    --velero-pod-cpu-request <CPU_REQUEST> \
+    --velero-pod-mem-request <MEMORY_REQUEST> \
+    --velero-pod-cpu-limit <CPU_LIMIT> \
+    --velero-pod-mem-limit <MEMORY_LIMIT> \
+    [--use-restic] \
+    [--restic-pod-cpu-request <CPU_REQUEST>] \
+    [--restic-pod-mem-request <MEMORY_REQUEST>] \
+    [--restic-pod-cpu-limit <CPU_LIMIT>] \
+    [--restic-pod-mem-limit <MEMORY_LIMIT>]
+```
+
+The values for the resource requests and limits flags follow the same format as [Kubernetes resource requirements][5]
+For plugin container images, please refer to our [supported providers][2] page.
+
+## Examples
+
+This section provides examples that serves as a starting point for more customized installations.
+
+```(bash)
+velero install --plugins velero/velero-plugin-for-gcp:v1.0.0 --bucket mybucket --provider gcp --secret-file ./gcp-service-account.json
```

Comment:
I like the pattern of always starting with (1) the `--provider` flag, then (2) the `--plugins` flag, which is pretty consistently followed elsewhere - so I'd do that for L36 and 38 here.

Reply:
copy-pasta!

Reply:
@ashish-amarnath looks like this comment still needs to be addressed